internal/http/retry.go

```diff
@@ -0,0 +1,59 @@
+package http
+
+import (
+	"fmt"
+	"task-runner-launcher/internal/logs"
+	"time"
+)
+
+const (
+	defaultMaxRetryTime           = 60 * time.Second // @TODO: What about long-running migrations?
```

Comment:
For discussion

Reply:
We could use the `/healthz/readiness` endpoint which doesn't return 200 until the DB is up and ready